### PR TITLE
Added defaultVerticalFieldOfView to XRRenderState

### PR DESCRIPTION
**Pull request by toji**
Pretty simple feature, based on the feedback that we got at the F2F. It's apparent to me that nobody is going to come up with a default value that has a good reason behind it, so I plugged 90deg in there just because it's a clean value that's easy to guess and gives reasonable results. Also left it as a single direction that can be specified (vertical) because you can pretty trivially do the math to convert from one to the other and vertical seems to be the value that most developers/game players are used to setting.

(FWIW I really hate the verbosity of the name, but I can see no reasonable way to shorten it without devolving into acronym soup like `defaultVFOV` 😛)

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -515,6 +515,23 @@ function drawScene() {
...
+If a different default field of view is desired, it can be specified by passing a new `defaultVerticalFieldOfView` value, in radians, to the `updateRenderState` method:
+
+```js
+// This changes the default vertical field of view to 0.4 radians (72 degrees).
+xrSession.updateRenderState({
+  defaultVerticalFieldOfView: 0.4 * Math.PI,
+});
+```
+
+The UA is allowed to clamp the value, and it is expected that if any physically-based values are available they should always be used in favor of the default.
```
[the remark this answers is not in the document]
Comment (toji) [reply explainer.md]: Sure, that sounds sensible to me.

**Comment by toji** [comment]
Updated to address feedback. PTAL!

File: explainer.md [hunk 2]
```diff
@@ -623,13 +640,15 @@ enum XREnvironmentBlendMode {
...
   double depthFar;
+  double verticalFieldOfView;
   XRLayer? baseLayer;
   XRPresentationContext? outputContext
 };
 
 [SecureContext, Exposed=Window] interface XRRenderState {
   readonly attribute double depthNear;
   readonly attribute double depthFar;
+  readonly attribute double defaultVerticalFieldOfView;
```
Comment (NellWaliczek) [inline explainer.md]: What about making this nullable?... Then we'd only populate it on inline sessions.  Either way, can we not call this "default" either?

**Comment by toji** [comment]
Moved to webxr/519

**Pull request closed without merging**